aiida/cmdline/commands/cmd_config.py

```diff
@@ -190,3 +192,30 @@ def verdi_config_caching(disabled):
                     echo.echo(identifier)
             elif disabled:
                 echo.echo(identifier)
+
+
+@verdi_config.command('upgrade')
+@click.argument('path', required=False, default=None, type=click.Path(exists=True, path_type=Path))
+@click.option('--version', type=int, default=None, help='Upgrade to specific version (default current).')
+def verdi_config_upgrade(version, path):
+    """Print a configuration, upgraded to a specific version."""
+    from aiida.manage.configuration import CURRENT_CONFIG_VERSION, get_config_path, upgrade_config
+    path = path or Path(get_config_path())
+    version = version if version is not None else CURRENT_CONFIG_VERSION
+    echo.echo_report(f'Upgrading configuration to v{version}: {path}')
+    config = json.loads(path.read_text(encoding='utf8'))
+    upgrade_config(config, version)
+    print(json.dumps(config, indent=2))
+
+
+@verdi_config.command('downgrade')
+@click.argument('version', type=int)
+@click.argument('path', required=False, default=None, type=click.Path(exists=True, path_type=Path))
+def verdi_config_downgrade(version, path):
+    """Print a configuration, downgraded to a specific version."""
+    from aiida.manage.configuration import downgrade_config, get_config_path
+    path = path or Path(get_config_path())
+    echo.echo_report(f'Downgrading configuration to v{version}: {path}')
+    config = json.loads(path.read_text(encoding='utf8'))
+    downgrade_config(config, version)
+    print(json.dumps(config, indent=2))
```

Comment:
this does not currently actually save the changes to disk.
Could do, obviously just extra code

Reply:
That is the behavior that I would expect though. Was your idea that people call this function and then manually copy past it into their config file? If that is the main use case, what is the advantage of making it indirect and forcing a user to perform a non-trivial manual operation?

Reply:
no I was just being lazy on the first parse lol, but will look into now implementing the save to disk

Reply:
done